sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/InputStreamContent.java

```diff
@@ -80,6 +80,11 @@ public Flux<ByteBuffer> toFluxByteBuffer() {
         return FluxUtil.toFluxByteBuffer(this.content, STREAM_READ_SIZE);
     }
 
+    @Override
+    public boolean isReplayable() {
+        return false;
```

Comment:
Should this be based on the type of input stream? If the source is ByteArrayInputStream, this should return true, for e.g.

Reply:
That's going to be part of https://github.com/Azure/azure-sdk-for-java/issues/28799